include/swift/Basic/APIntMap.h

```diff
@@ -38,11 +38,11 @@ struct WidthPreservingAPIntDenseMapInfo {
   }
 
   static unsigned getHashValue(const APInt &Key) {
-    return static_cast<unsigned>(hash_value(Key));
+    return llvm::DenseMapInfo<APInt>::getHashValue(Key);
```

Comment:
@rjmccall any objections to this change? I'm actually not sure what header is no longer including `Hashing.h` but this avoids including any extra entirely 🤷